Tools-Override/tests.targets

```diff
@@ -82,10 +89,14 @@
       <RunTestsForProjectInputs Include="@(_DebugSymbolsIntermediatePath)" />
       <RunTestsForProjectInputs Include="@(AllItemsFullPathWithTargetPath)" />
     </ItemGroup>
-    <ItemGroup>
+    <ItemGroup Condition="'$(RunningOnNetFx)' != 'true'">
       <SupplementalTestData Include="$(XunitRuntimeConfig)" />
       <SupplementalTestData Include="$(RuntimePath)xunit.console.netcore.exe" />
     </ItemGroup>
+    <ItemGroup Condition="'$(RunningOnNetFx)' == 'true'">
+      <SupplementalTestData Include="$(RuntimePath)xunit*.*" />
```

Comment:
As we chatted I would still like to have an issue tracking whether or not we can get away with having the runner stay in a shared location for desktop. That would be preferable to having to copy these into every test output directory. Perhaps something as simple as having an app.config that adds an additional probing path would work.